templates/splash/splashpage.md

```diff
@@ -10,35 +10,11 @@ subject-en: Government of Canada, services
 subject-fr: Gouvernement du Canada, services
 dateModified: 2020-06-03
```

Comment:
Should this be updated?

Reply:
Not really, the content has not change, only moved